data/common.yaml

```diff
@@ -1,8 +1,2 @@
 ---
-influxdb::host: "%{facts.networking.fqdn}"
-influxdb::port: 8086
-influxdb::initial_org: 'puppetlabs'
-influxdb::initial_bucket: 'puppet_data'
-influxdb::repo_gpg_key_id: '9D539D90D3328DC7D6C8D3B9D8FF8E1F7DF8B07E'
-influxdb::repo_gpg_key_url: 'https://repos.influxdata.com/influxdata-archive_compat.key'
-influxdb::manage_repo: false
+{}
```

Comment:
my personal preference is to delete the file. When I see it in the filesystem I always assume it also contains data.

Reply:
I've seen warnings in the puppetserver log about missing common.yaml, maybe it only does that for the control repo hiera?

Reply:
that sounds odd. It shouldn't complain about it when the file doesn't exist. Only when the file isn't valid yaml. Often it's really empty or only contains `---`, both aren't correct.